onnxruntime/core/optimizer/transpose_optimizer/optimizer_api.h

```diff
@@ -61,6 +61,10 @@ enum class DataType : int32_t {
   COMPLEX64 = 14,
   COMPLEX128 = 15,
   BFLOAT16 = 16,
+  FLOAT8E4M3FN = 17,
+  FLOAT8E4M3FNUZ = 18,
+  FLOAT8E5M2 = 19,
+  FLOAT8E5M2FNUZ = 20,
```

Comment:
Is there any way to avoid this enum repetition?

This is the same language, it should be possible to put it into one header?

Reply:
I would prefer to do it in another PR as it should probably include all types and not only float 8 types.